harness/determined/tensorboard/metric_writers/pytorch.py

```diff
@@ -2,14 +2,18 @@
 from typing import Any, Union
 
 import numpy as np
+import torch
+from packaging import version
 
 from determined import tensorboard
 
 # As of torch v1.9.0, torch.utils.tensorboard has a bug that is exposed by setuptools 59.6.0.  The
 # bug is that it attempts to import distutils then access distutils.version without actually
 # importing distutils.version.  We can workaround this by prepopulating the distutils.version
 # submodule in the distutils module.
-import distutils.version  # isort:skip  # noqa: F401
+if version.parse("1.9.0") <= version.parse(torch.__version__) < version.parse("1.11.0"):
```

Comment:
this if statement seems unnecessary.  Though it's also harmless.

Reply:
(I mean the `if` conditional itself, vs just always forcing the import)

Reply:
I think it's fine to include. We can stay consistent with this philosophy: https://github.com/determined-ai/determined/pull/7515#discussion_r1279935915